ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ReportJobs.tsx

```diff
@@ -76,22 +79,43 @@ function ReportJobs({ scanConfig, isComplianceReportingEnabled }: ReportJobsProp
     const onReportStatesFilterChange = (_checked: boolean, selectedStatus: RunState) => {
         const isStatusIncluded = filteredReportRunStates.includes(selectedStatus);
         if (isStatusIncluded) {
+            const newFilters = ensureReportRunStates(
```

Comment:
Although off the subject of this contribution, the repetition brings question to mind:
Is it possible to factor out into one assignment, `analyticsTrack` call, and `setSearchFilter` call?

```ts
const newFilters = ensureReportRunStates(
    isStatusIncluded ? filteredReportRunStates.filter(…) : [...filteredReportRunStates, selectedStatus]  
);
// analyticsTrack

@bradr5
// setSearchFilter
```